app/views/categories/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="navigation">
+  <ul>
+    <li><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+    <h2><%= @category.name %></h2>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link" %></li>
+  </ul>
+</div>
+<div class="category-header">
+  <p><%= @category.name %></p>
+  <p>Total Amount: <%= @category.purchases.sum(:amount) %></p>
+</div>
+<div class="purchases-container">
+
+  <% @category.purchases.each do |purchase| %>
+    <div class="purchase-item">
+      <%= link_to purchase.name, category_purchase_path(@category, purchase) %>
+      - Amount: <%= purchase.amount %>
+    </div>
+  <% end %>
+</div>
+<div class="new-category">
+  <div class="new-category-inner">
+    <%= link_to "Add a New Purchase", new_category_purchase_path(@category) %>
+  </div>
+</div>
```

Comment:
- [ ] Based on the requirements, there should be four core pages apart from the _**Splash**_ Page, the _**Signup**_ and _**Login**_ Pages. The pages are as follows:

- A categories index Page that displays all categories
- A form page for adding new categories
- The transaction index page that displays details of a transaction when you click an item on the categories index page. 
- The form page for adding a transaction and mapping it to a category or categories.

Kindly design the movement between pages for users based on the four pages described above. 👍 😎